src/components/forms/OnboardingRegistrationForm.tsx

```diff
@@ -0,0 +1,85 @@
+"use client";
+import { genders } from "@/shared/onboarding";
+import { formSchema } from "@/shared/onboarding";
+import { FormSchema } from "@/shared/onboarding";
+import { useForm } from "react-hook-form";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { Input, Label, Select, Form, Title } from "@/shared/components";
+
+export default function OnboardingRegistrationForm() {
+  const {
+    register,
+    handleSubmit,
+    watch,
+    formState: { errors },
+  } = useForm<FormSchema>({
+    resolver: zodResolver(formSchema),
```

Comment:
Dont we need to breakdown the schema first?

Reply:
Actually it was working, but as we talked I made the changes. Thanks.